docs/features/onboarding.md

```diff
@@ -0,0 +1,61 @@
+# WooCommerce Onboarding
+
+The onboarding feature is a reimagined new user setup experience for WooCommerce core. It contains two sections aimed at getting merchants started with their stores. The merchant begins with the "profile wizard", which helps with initial steps like setting a store address, making extension recommendations, and connecting to Jetpack for additional features. Once the profile wizard is complete, the merchant can purchase & install optional extensions via WooCommerce.com, before continuing to the "task list". From the task list, merchants are walked through a variety of items to help them start selling, such as adding their first product and setting up payment methods.
+
+* Master Thread: p6q8Tx-R1-p2
+* Designs: 2317a-pb
+
+## Enabling Onboarding
+
+If you run the development version of WooCommerce Admin from GitHub directly, no further action should be needed to enable Onboarding.
+
+Users of the published WooCommerce Admin plugin need to either opt-in to using the new onboarding experience manually, or become part of the a/b test in core. See https://github.com/woocommerce/woocommerce/pull/24991 for more details on the testing implementation.
+
+To enable the new onboarding experience manually, log-in to `wp-admin`, and go to `WooCommerce > Settings > Help > Setup Wizard`. Click `Enable` under the `New onboarding experience` heading.
+
+You can also set the following configuration flag in your `wp-config.php`:
+
+`define( 'WOOCOMMERCE_ADMIN_ONBOARDING_ENABLED', true );`
+
+## New REST API endpoints
+
+@todo
+
+## Onboarding filters and hooks
+
+@todo
+
+## Options and settings
+
+@todo
+
+## WCCOM connection and redirection
```

Comment:
Since this is for public consumption maybe `WooCommerce.com connection` would be a better heading.